microcosm_flask/tests/test_paging.py

```diff
@@ -116,3 +116,56 @@ def search_foo():
                 },
             }
         })))
+
+
+def test_custom_paginated_list():
+    graph = create_object_graph(name="example", testing=True)
+    ns = Namespace(subject="foo", object_="bar")
+
+    class CustomPage(Page):
```

Comment:
Here's an example of customized paging done generically. I'm contemplating moving this `@classmethod` as the default. This would mean that any arguments passed in the query string (as defined by the filter schema) would automatically end up in the pagination expansion. This is usually what we want when filtering down a search, but I was reluctant to make it the default behavior without some discussion.